src/main/java/com/exadel/coolDesking/bot/config/botConfig.java

```
@@ -0,0 +1,42 @@
+package com.exadel.coolDesking.bot.config;
+
+import com.exadel.coolDesking.bot.HotDeskingBot;
+import com.exadel.coolDesking.bot.UpdateReceiver;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.api.methods.updates.SetWebhook;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
+import org.telegram.telegrambots.updatesreceivers.DefaultWebhook;
+
+@Configuration
+public class botConfig {
+    private final String WEBHOOK = System.getenv("WEBHOOK");
+    private final String URL = System.getenv("URL");
+
+    @Bean
+    public SetWebhook setWebhookInstance() {
+        return SetWebhook.builder()
+                .url(WEBHOOK)
+                .build();
+    }
+
+    @Bean
+    public HotDeskingBot HotDeskingSpringWebhookBot(
+            SetWebhook setWebhookInstance,
+            UpdateReceiver updateReceiver
+    ) {
+        HotDeskingBot hotDeskingBot = new HotDeskingBot(setWebhookInstance, updateReceiver);
+        DefaultWebhook defaultWebhook = new DefaultWebhook();
+        defaultWebhook.setInternalUrl(URL);
+        try {
+            TelegramBotsApi telegramBotsApi = new TelegramBotsApi(DefaultBotSession.class);
+            telegramBotsApi.registerBot(hotDeskingBot, setWebhookInstance);
+            return hotDeskingBot;
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+            return null;
```

maybe you want to fail the application startup if the bot can't be loaded ? You need to rethrow the exception then.